eacl/validator.go

```diff
@@ -1,136 +0,0 @@
-package eacl
-
-import (
-	"bytes"
-)
-
-// Validator is a tool that calculates
-// the action on a request according
-// to the extended ACL rule table.
-type Validator struct {
-}
-
-// NewValidator creates and initializes a new Validator using options.
-func NewValidator() *Validator {
-	return &Validator{}
-}
-
-// CalculateAction calculates action on the request according
-// to its information represented in ValidationUnit.
-//
-// The action is calculated according to the application of
-// eACL table of rules to the request.
-//
-// Second return value is true iff the action was produced by a matching entry.
-//
-// If no matching table entry is found or some filters are missing,
-// ActionAllow is returned and the second return value is false.
-func (v *Validator) CalculateAction(unit *ValidationUnit) (Action, bool) {
```

Comment:
we are so get used to moving code from one repo to another and many other types of useless work. am not a conservative but can we at least explain why we moving smth in and out again?